.storybook/main.js

```diff
@@ -12,4 +12,25 @@ module.exports = {
     ),
     "@storybook/addon-docs",
   ],
+  babel(babelConfig) {
+    return {
+      ...babelConfig,
+      presets: babelConfig.presets.map((preset) => {
+        const presetName = Array.isArray(preset) ? preset[0] : preset;
+        const presetOptions = Array.isArray(preset) ? preset[1] : [];
+
+        if (presetName !== require.resolve("@babel/preset-react")) {
+          return preset;
+        }
```

Comment:
NBD
Could return immediately, makes it a little more obvious this is only for preset-react
```suggestion
        if (presetName !== require.resolve("@babel/preset-react")) {
          return preset;
        }

        const presetName = Array.isArray(preset) ? preset[0] : preset;
        const presetOptions = Array.isArray(preset) ? preset[1] : [];
```

Reply:
This will use `presetName` before it's defined; what's the advantage of this?

Reply:
🤦 misread that haha, disregard!